ethexe/db/src/database.rs

```diff
@@ -478,6 +478,24 @@ impl CodesStorageWrite for Database {
         self.kv
             .put(&Key::CodeValid(code_id).to_bytes(), valid.encode());
     }
+
+    fn valid_codes(&self) -> BTreeSet<CodeId> {
```

Comment:
agree with @grishasobol review: 
we may rename Key::CodeValid into Key::CodeValidationStatus and fn valid_codes will filter only actually Valid (validation status == true) codes